### PR TITLE
bruteforce

### PR DESCRIPTION
**Pull request by jameskimmel**
### ☑️ Resolves

* Brute force is enabled by default, but not the app.

### CODE REVIEW

File: admin_manual/configuration_server/bruteforce_configuration.rst [hunk 1]
```diff
@@ -72,14 +72,14 @@ Activating
 ~~~~~~~~~~
 
 Brute force protection is enabled by default on Nextcloud. Its behavior can be adjusted through the 
-``bruteforcesettings`` app (shipped with Server and enabled by default), several ``occ`` commands, and several 
+``bruteforcesettings`` app, several ``occ`` commands, and several
```
Comment (joshtrichards) [inline admin_manual/configuration_server/bruteforce_configuration.rst:75]:
```suggestion
``bruteforcesettings`` app (shipped with Server, but disabled by default), several ``occ`` commands, and several 
```

File: admin_manual/configuration_server/bruteforce_configuration.rst [hunk 2]
```diff
@@ -72,14 +72,14 @@ Activating
...
-``bruteforcesettings`` app (shipped with Server and enabled by default), several ``occ`` commands, and several 
+``bruteforcesettings`` app, several ``occ`` commands, and several 
 ``config.php`` parameters. Its effectiveness is highly dependent on having a properly configured environment, 
 particularly when integrating a reverse proxy with Nextcloud (and associated parameters such as ``trusted_proxies``).
 
 The brute force settings app
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This app, which shipped and enabled by default, makes it possible (via the Web UI) to view the status of a 
+This app makes it possible (via the Web UI) to view the status of a
```
Comment (joshtrichards) [inline admin_manual/configuration_server/bruteforce_configuration.rst:82]:
```suggestion
This app (once enabled) makes it possible (via the Web UI) to view the status of a 
```

**Review by joshtrichards — COMMENTED** [review]
Good catch!

**Comment by joshtrichards** [comment]
Though maybe we should have this enabled by default anyhow. :-) Either way, docs should be updated to reflect current behavior. Thanks for sending this in!

**Comment by jameskimmel** [comment]
You are very welcome, and thank you for taking a look. I enabled it today for testing but disabled it again since I don't see any use case for me. Is there really no setting besides the whitelist.

**Comment by nickvergessen** [comment]
Since the app is now shipped and enabled by default in 30+ close this and send it against 29?

**Comment by jameskimmel** [comment]
> Since the app is now shipped and enabled by default in 30+ close this and send it against 29?

Sorry, I don't know how stuff like this works here. Should I close the issue? 
What do you mean be "send it against 29"?

**Comment by nickvergessen** [comment]
No problem, I can do that for you

**Comment by nickvergessen** [comment]
https://github.com/nextcloud/documentation/pull/12118

**Pull request closed without merging**